docs/specification/1.0.md

```diff
@@ -241,6 +242,8 @@ Field | Description
 <nobr>`label`</nobr>| *string*. human-readable label to display for this link (e.g. the EHR might render this as the underlined text of a clickable link).
 `url` | *URL*. URL to load (via `GET`, in a browser context) when a user clicks on this link. Note that this may be a "deep link" with context embedded in path segments, query parameters, or a hash.
 `type` | *string*. The type of the given URL. There are two possible values for this field. A type of `absolute` indicates that the URL is absolute and should be treated as-is. A type of `smart` indicates that the URL is a SMART app launch URL and the EHR should ensure the SMART app launch URL is populated with the appropriate SMART launch parameters.
+<nobr>`appContext`</nobr> |*string*.  an optional field to share addtional app context from CDS Service to EHR. The `appContext` field should only be valued if the link type is `smart` and is not valid for `absolute` links. The `appContext` field and value will be sent to the SMART app alongside the other launch context when the SMART app is launched.
```

Comment:
I think it would be more clear to wordsmith the first sentence to (also fixed a typo):

"An optional field that allows the CDS Service to pass context regarding the launch of this SMART app from the CDS card to the SMART app."

Also, you don't need the `<nobr>` tags around `appContext`

Reply:
I like your version. I have updated text based on your input. 

as for <nobr> , there are many places in this file which leverages this tag to prevent wrap .  I have removed this based on your comment but something to thing about